homeassistant/components/system_bridge/__init__.py

```diff
@@ -183,6 +209,18 @@ async def handle_send_text(call: ServiceCall) -> None:
         ),
     )
 
+    hass.services.async_register(
+        DOMAIN,
+        SERVICE_POWER_COMMAND,
+        handle_power_command,
+        schema=vol.Schema(
+            {
+                vol.Required(CONF_BRIDGE): valid_device,
+                vol.Required(CONF_COMMAND): valid_power_command,
```

Comment:
I can see why valid_device needed a checking function, but couldn't valid_power_command be done with a normal voluptuous "in list" type pattern?

I assume that would be better for any GUI that might use it too.

Reply:
Could you use 'in list'?

https://developers.home-assistant.io/docs/development_validation/#lists